pkg/controllers/federatedcluster/clusterjoin.go

```diff
@@ -309,10 +311,10 @@ func createAuthorizedServiceAccount(
 	if err != nil {
 		return "", fmt.Errorf("error creating service account token secret %s : %w", MemberServiceAccountName, err)
 	}
-	logger.Info(fmt.Sprintf("Created service account token secret %s for service account %v", saTokenSecretName, MemberServiceAccountName))
+	logger.V(1).WithValues("sa-toke-secret-name", saTokenSecretName).Info("Created service account token secret for service account")
```

Comment:
```suggestion
	logger.V(1).WithValues("sa-token-secret-name", saTokenSecretName).Info("Created service account token secret for service account")
```
Let's add this key-value pair to the logger before we create the service account token secret.